lib/github-glue.ts

```diff
@@ -91,6 +91,32 @@ export class GitHubGlue {
         return checks.data.id;
     }
 
+    /**
+     * Add a cc to a Pull Request
+     *
+     * @param {string} pullRequestURL the Pull Request to comment on
+     * @param {string} cc to add
+     * @returns the comment ID and the URL to the comment
+     */
+    public async addPRcc(pullRequestURL: string, cc: string):
+        Promise<void> {
+        const id = cc.match(/(<.*>)/);
+
+        if (id) {                   // filter out gitster@pobox.com?
+            const url = GitGitGadget.parsePullRequestURL(pullRequestURL);
+            const pr = await this.getPRInfo(url[0], url[2]);
+            const ccNow = pr.body.match(/\n\n([Cc][Cc]:.*)/s);
```

Comment:
Does the `i` attribute work? It seems to work over here:

```sh
$ node -e 'console.log("Cc:".match(/cc/i));'
[ 'Cc', index: 0, input: 'Cc:', groups: undefined ]
```

Reply:
I cannot explain why I did not use it.  Updating.